frontend/src/scenes/insights/utils/cleanFilters.ts

```diff
@@ -136,7 +136,9 @@ export function cleanFilters(filters: Partial<FilterType>, oldFilters?: Partial<
             ...filters,
             interval: autocorrectInterval(filters),
             display:
-                filters.session && filters.session === 'dist'
+                filters.breakdown && !oldFilters?.breakdown
```

Comment:
This `cleanFilters` can be called from anywhere, and `oldFilters` might not always be there. Right now if `oldFilters` doesn't exist, the function will switch any chart with a breakdown to a bar chart. I don't know what side-effects it has, but just in case I'd add `&& oldFilters` to the if to avoid finding out.